### PR TITLE
Fix calculation of number of app screens

### PR DESCRIPTION
**Pull request by mark9064**
Was testing an extra app (so 12 in total) and noticed the extra screen added
Also removed the old comment

There might be a nicer C++ way to do this than this macro I spotted lying around, but I can't find one

### CODE REVIEW

File: src/displayapp/screens/ApplicationList.h [hunk 1]
```diff
@@ -38,8 +38,7 @@ namespace Pinetime {
 
         static constexpr int appsPerScreen = 6;
 
-        // Increment this when more space is needed
-        static constexpr int nScreens = (UserAppTypes::Count / appsPerScreen) + 1;
+        static constexpr int nScreens = CEIL_DIV(UserAppTypes::Count, appsPerScreen);
```
Comment (FintasticMan) [inline src/displayapp/screens/ApplicationList.h:41]: I'm not sure where this macro comes from. I would prefer using `(UserAppTypes::Count - 1) / appsPerScreen + 1`. You should also remove the comment above as it no longer makes sense.

**Review by FintasticMan — CHANGES_REQUESTED** [review]
Good find.

File: src/displayapp/screens/ApplicationList.h (hunk 1 above)
Comment (mark9064) [reply src/displayapp/screens/ApplicationList.h:41]: Beaten you to it with the comment, it's already gone :)
The macro is from the NRF SDK. Personally I think it's clearer semantically but I'll change it to what you suggest

Comment (FintasticMan) [reply src/displayapp/screens/ApplicationList.h]: Thank you! While I do think it is clearer what it's doing with the macro, I don't think that it makes sense to use the nRF SDK here.

**Review by FintasticMan — APPROVED** [review]

File: src/displayapp/screens/ApplicationList.h (hunk 1 above)
Comment (mark9064) [reply src/displayapp/screens/ApplicationList.h]: That's a valid point, probably not a great dependency for this file

**Review by JF002 — APPROVED** [review]

**Comment by JF002** [comment]
@mark9064  Good catch, thanks!

**Pull request merged**